src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -190,24 +196,29 @@ public void feederMain() {
         } else if (oi.reverseIntake()) {
             startFeeder(-feederSpeed);
         } else {
+            gathererState = gathererUp;
             holdFeeder();
         }
     }
 
     public void startFeeder(double speed) {
         RESET_ENCODER = true;
         feederPid.setReference(speed, ControlType.kDutyCycle);
+        gathererState = gathererDown;
+
     }
 
     public void holdFeeder() {
         if (RESET_ENCODER) {
             feederEncoder.setPosition(0);
             RESET_ENCODER = false;
         }
+        gathererState = gathererUp;
         feederPid.setReference(0, ControlType.kPosition);
     }
 
     public void stopFeeder() {
+        gathererState = gathererUp;
```

Comment:
same here - rely on the driver to raise the intake?

Reply:
fixed